wikimedia/wikipedia-ios
# Native Recommendations - add translations

**Phabricator:** https://phabricator.wikimedia.org/T363925

### Notes
Adds a localized string that didn't come through Translate Wiki in the latest PR. Spanish came through; I just edited the dash.

### Test Steps
1. On Image recs, go to "Problem with feature" in the overflow menu in Chinese, Portuguese, Spanish or French . Confirm "[Screenshots or Links]" is added at the end in the correct language.
